sky/skylet/providers/lambda_cloud/node_provider.py

```diff
@@ -164,31 +170,39 @@ def internal_ip(self, node_id: str) -> str:
     def create_node(self, node_config: Dict[str, Any], tags: Dict[str, str],
                     count: int) -> None:
         """Creates a number of nodes within the namespace."""
-        assert count == 1, count  # Only support 1-node clusters for now
-
-        # get the tags
+        # Get tags
         config_tags = node_config.get('tags', {}).copy()
         config_tags.update(tags)
         config_tags[TAG_RAY_CLUSTER_NAME] = self.cluster_name
 
-        # create the node
+        # Create nodes
         ttype = node_config['InstanceType']
```

Comment:
Would be nice to use fullname for the variable `instance_type`?

Reply:
Done.